`[error] FileNotFoundError: [Errno 2] No such file or directory: '/run/ipa/ccaches'`

I observe failures
```
Configuring the web interface (httpd)
  [1/21]: stopping httpd
  [2/21]: backing up ssl.conf
  [3/21]: configuring mod_ssl certificate paths
  [4/21]: setting mod_ssl protocol list
  [5/21]: configuring mod_ssl log directory
  [6/21]: disabling mod_ssl OCSP
  [7/21]: adding URL rewriting rules
  [8/21]: configuring httpd
  [9/21]: setting up httpd keytab
  [10/21]: configuring Gssproxy
  [11/21]: setting up ssl
  [12/21]: configure certmonger for renewals
  [13/21]: publish CA cert
  [14/21]: clean up any existing httpd ccaches
  [error] FileNotFoundError: [Errno 2] No such file or directory: '/run/ipa/ccaches'
FreeIPA server configuration failed.
tests/run-in-k8s.sh: line 17: kill: (7223) - No such process
+ kill 7223
+ :
+ trap - EXIT
Error: Process completed with exit code 1.
```
with AlmaLinux 9 and Rocky Linux 9 on Kubernetes setups with CRI-O, for example https://github.com/freeipa/freeipa-container/actions/runs/31996277844/job/95288987506.

The problem is not just with newly built images but with the images that are already in registries as well, so it is not immediately clear if the problem is caused by some change in RHEL 9 / AlmaLinux 9 / Rocky Linux 9, or some change in CRI-O or some other component.

We don't see the problem with non-Kubernetes podman or docker tests.

We only saw the problem on Ubuntu 24.04 host but that is only because https://github.com/freeipa/freeipa-container/issues/737 still has not resolved, so we don't have any data from Kubernetes on Ubuntu 26.04.

--- a/.github/actions/install-crio/action.yaml
+++ b/.github/actions/install-crio/action.yaml
@@ -18,5 +18,12 @@ runs:
         # for images like rancher/local-path-provisioner:v0.0.31 
         # due to multiple unqualified-search-registries entries in /etc/containers/registries.conf.d/crio.conf
         echo 'unqualified-search-registries = ["docker.io"]' | sudo tee /etc/containers/registries.conf.d/no-unqualified-quay.conf
+
+        # Workaround https://github.com/cri-o/cri-o/issues/9865 / https://github.com/freeipa/freeipa-container/issues/750
+        if [ -f /etc/containers/storage.conf ] ; then
+            sudo mv -f /etc/containers/storage.conf /etc/containers/storage.conf.dist
+            sudo sudo podman system reset -f
+        fi
+
         sudo systemctl start crio.service
       shell: bash -euxo pipefail {0}

--- a/.github/build-test-params.yaml
+++ b/.github/build-test-params.yaml
@@ -89,8 +89,6 @@ k8s:
       runtime: cri-o
     - runs-on: ubuntu-26.04
       runtime: containerd
-    - runs-on: ubuntu-24.04
-      runtime: cri-o
 
 push:
   exclude:

